src/state/slices/tradeQuoteSlice/selectors.ts

```diff
@@ -585,24 +588,47 @@ export const selectActiveQuoteAffiliateBps: Selector<ReduxState, string | undefi
     return activeQuote.affiliateBps
   })
 
-export const selectQuoteAffiliateFeeUserCurrency = createSelector(
-  selectActiveQuote,
-  selectQuoteSellAmountUserCurrency,
+type AffiliateFeesProps = {
+  feeModel: ParameterModel
+  inputAmountUsd: string | undefined
+}
+
+export const selectCalculatedFees: Selector<ReduxState, CalculateFeeBpsReturn> = createSelector(
```

Comment:
this needs to be `createCachedSelector` using the `feeModel` and `inputAmountUsd` as cache keys

Reply:
since quote and input are both "trade"

Reply:
Might be best to put this in common selectors since its not strictly trade related

Reply:
Cached in https://github.com/shapeshift/web/pull/6503/commits/489a1bec73fbeaf28858971c29bbdb981f783f5b

Moved to common in https://github.com/shapeshift/web/pull/6503/commits/0cee38bca30565e90074eba43e3a21ed6acdc920